platform/vendor/vendor_tekton_chains.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# NOTE: This is based on the release validation process for tekton pipelines: https://github.com/tektoncd/pipeline/releases
+RELEASE_FILE=https://storage.googleapis.com/tekton-releases/chains/previous/v0.5.0/release.yaml
+REKOR_UUID=3a62d47dcbe0727513ac2e2dcc3a41bfd413ebc128bb661ed4f115d4db83200f
+
+TEKTON_DIR=tekton/chains
+
+# Obtains the list of images with sha from the attestation
+REKOR_ATTESTATION_IMAGES=$(rekor-cli get --uuid "$REKOR_UUID" --format json | jq -r .Attestation | base64 --decode | jq -r '.subject[]|.name + ":v0.5.0@sha256:" + .digest.sha256')
+
```

Comment:
Nitpicking  little, and not a blocking comment at all, but could/should we extract the version (`v0.5.0`) into its own variable, and use the variable instead? This comment would apply to line 6 as well.

Reply:
Yep, I saw the same thing in both this and pipeline vendoring. I think once @trmiller messes with his vendorme tool a bit it will go away anyway. I created: #53 to solve this issue.